bitnami/kibana/templates/deployment.yaml

```diff
@@ -0,0 +1,188 @@
+          {{- end }}
+          {{- if (include "kibana.importSavedObjects" .) }}
+          - name: saved-objects-init-scripts
+            mountPath: /docker-entrypoint-initdb.d/saved-objects-import
+          {{- end }}
+          {{- if .Values.savedObjects.configmap }}
+          - name: saved-objects-configmap
+            mountPath: /bitnami/kibana/saved-objects
+          {{- end }}
+          {{- if .Values.initScriptsCM }}
+          - name: custom-init-scripts-cm
+            mountPath: /docker-entrypoint-initdb.d/cm
+          {{- end }}
+          {{- if .Values.initScriptsSecret }}
+          - name: custom-init-scripts-secret
+            mountPath: /docker-entrypoint-initdb.d/secret
+          {{- end }}
+          {{- if .Values.extraVolumeMounts }}
+          {{- tpl (toYaml .Values.extraVolumeMounts) $ | nindent 6 }}
+          {{- end }}
+      {{- if .Values.sidecars }}
+      {{- tpl (toYaml .Values.sidecars) $ | nindent 6 }}
+      {{- end }}
+      volumes:
+        - name: kibana-data
+        {{- if .Values.persistence.enabled }}
+          persistentVolumeClaim:
+            claimName: {{ include "kibana.pvc" . }}
+        {{- else }}
+          emptyDir: {}
+        {{ end }}
+        - name: kibana-config
+          configMap:
+            name: {{ include "kibana.configurationCM" . }}
+        {{- if (include "kibana.importSavedObjects" .) }}
+        - name: saved-objects-init-scripts
+          configMap:
+            name: {{ include "kibana.fullname" . }}-saved-objects
+            defaultMode: 0755
+        {{- end }}
+        {{- if .Values.plugins }}
+        - name: plugins-init-scripts
+          configMap:
+            name: {{ include "kibana.fullname" . }}-plugins
+            defaultMode: 0755
+        {{- end }}
+        {{- if .Values.initScriptsCM }}
+        - name: custom-init-scripts-cm
+          configMap:
+            name: {{ template "kibana.initScriptsCM" . }}
+            defaultMode: 0755
+        {{- end }}
+        {{- if .Values.initScriptsSecret }}
+        - name: custom-init-scripts-secret
+          secret:
+            name: {{ template "kibana.initScriptsSecret" . }}
+            defaultMode: 0755
+        {{- end }}
+        {{- if .Values.savedObjects.configmap }}
+        - name: saved-objects-configmap
+          configMap:
+            name: {{ template "kibana.savedObjectsCM" . }}
+        {{- end }}
+      {{- if .Values.extraVolumes }}
+      {{- tpl (toYaml .Values.extraVolumes) $ | nindent 6 }}
+      {{- end }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector:
+      {{- tpl (toYaml .Values.nodeSelector) $ | nindent 6 }}
+      {{- end }}
+      {{- if .Values.affinity }}
+      affinity:
+      {{- tpl (toYaml .Values.affinity) $ | nindent 6 }}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations:
+      {{- tpl (toYaml .Values.tolerations) $ | nindent 6 }}
+      {{- end }}
```

Comment:
`tolerations`, `affinity` and `nodeSelector` should be evaluated as a template. See other charts.
Remember to reflect the changes in the **README.md** and **values.yaml**/**values-production.yaml**

Reply:
I'm confused. The three are being evaluated as templates

Reply:
Oh you're right! Could you please reflect it in the README.md and the associated comments in the **values.yaml**?

Reply:
Done